tests/src/Functional/CommerceBrowserTestBaseTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\Tests\commerce\Functional;
+
+/**
+ * Tests CommerceBrowserTestBase functionality.
+ *
+ * @group commerce
+ */
+class CommerceBrowserTestBaseTest extends CommerceBrowserTestBase {
```

Comment:
This test feels a bit too meta for me. I know 1.x did it too, but it doesn't make much sense. If the code for creating a user or an entity fails, then literally every test will fail, so we'll know that the test base is broken.